src/server.js

```diff
@@ -13,7 +15,10 @@ app.use(express.json());
 app.use(cors());
 
 // routes
-app.use('/api', apiRoutes);
+app.use('/auth', userAuth);
+// Free routes will go here e.g. html routes
+// Authenticate users here
```

Comment:
One thing you might find out and about in code bases is: //TODO: Implement the user authentication middleware [ticket name]